docs/report_utils.md

```diff
@@ -0,0 +1,178 @@
+GPLT Report Utilities
+=========================
+
+# Background
+These scripts were used in the process of creating the Kearney soak test report.
+They are a work-in-progress. 
+Some manual steps are still required, but this gets us closer to a fully automated process.
+
+## Gatling Reports
+Gatling produces a detailed HTML report from the simulation.log file. 
+However, our Apples to Apples test reports include a summarized comparison of the data from two test runs.
+Producing this report currently requires manually copying the data from each HTML report into the corresponding table in a Google Sheets doc.
+Future possibilities include using the data in BigQuery rather than relying on CSV files (see https://github.com/puppetlabs/gatling-puppet-load-test/pull/229#discussion_r188406981).
+
+Notes: 
+* The following examples were created using previous runs. 
+GPLT currently organizes the results directories based on the hostname; adjust actual paths accordingly.
+* The scripts should be run from the `util/report_utils` directory as shown in the examples below.
+
+### examples.tar.gz
+Extract the `examples.tar.gz` archive to use the provided CSV, HTML, and image files in the examples below:
+```
+test.user:~/gatling-puppet-load-test/util/report_utils> tar xzf examples.tar.gz
+```
+
+### extract_csv.rb
+This script parses the JSON results file for a specified test run and creates a CSV file with the data used in our Apples to Apples test report.
+
+Run `extract_csv.rb` by providing the full path to the result folder; for example:
+
+```
+test.user:~/gatling-puppet-load-test/util/report_utils> ruby extract_csv.rb /Users/test.user/gatling-puppet-load-test/results/perf/PerfTestLarge-1557765424829
```

Comment:
These result directories from previous runs aren't actually included in the examples tarball. I'm going to update it to include them and update the steps to reference them rather than the directories referenced below.